config/config.mk

```diff
@@ -30,7 +30,7 @@ boot_addr ?= A0000000
 
 # L2 memory configuration (in hex)
 l2_base  ?= 80000000
-l2_size  ?= 10000
```

Comment:
I get that this is a very small L2, but any reason why we need to change the default size in this MR?